test/integration/functional_test.go

```diff
@@ -193,9 +193,10 @@ func validateLoadImage(ctx context.Context, t *testing.T, profile string) {
 	// make sure the image was correctly loaded
 	var cmd *exec.Cmd
 	if ContainerRuntime() == "docker" {
-		cmd = exec.CommandContext(ctx, Target(), "ssh", "-p", profile, "--", "docker", "images", "--format", "{{.Repository}}:{{.Tag}}")
+		cmd = exec.CommandContext(ctx, Target(), "ssh", "-p", profile, "--", "docker", "image", "inspect", newImage)
 	} else {
-		cmd = exec.CommandContext(ctx, Target(), "ssh", "-p", profile, "--", "sudo", "ctr", "-n=k8s.io", "image", "ls")
+		// crictl inspecti localhost/busybox:test-example
+		cmd = exec.CommandContext(ctx, Target(), "ssh", "-p", profile, "--", "sudo", "crictl", "inspecti", "localhost/"+newImage)
```

Comment:
Not sure if containerd adds the localhost/ prefix, I think that is a podman thing. Might have to strip it out ?

Alternatively make sure to use qualified images everywhere, such as "docker.io/busybox". And re-add it later.

Reply:
good catch